src/DomainServices/Services/Interfaces/ICustomerService.cs

```diff
@@ -1,22 +1,24 @@
 using System;
 using System.Collections.Generic;
-using DesafioWarren.WebAPI.Models;
+using DomainModels;
 
-namespace DesafioWarren.WebAPI.Services;
+namespace DomainServices.Services;
 
 public interface ICustomerService
-{    
-    bool CreateCustomer(Customer customer);
+{
+    (bool isValid, string message) CreateCustomer(Customer customer);
 
     IList<Customer> GetCustomers();
 
-    Customer GetCustomerByEmail(string Email);
+    bool CustomerForCpfAlreadyExists(string cpf);
+
+    bool CustomerForEmailAlreadyExists(string email);
```

Comment:
Usar o padrão da microsoft de nomenclaturas

```suggestion
    bool AnyCustomerForCpf(string cpf);

    bool AnyCustomerForEmail(string email);
```